README.md

```diff
@@ -22,6 +22,8 @@ The requirements can also be installed with:
 ```bash
 pip install -r requirements.txt
 ```
+We need `mmd` from [github.com/metno/mmd](https://github.com/metno/mmd).
+We use path to `mmd-to-geonorge.xsl` and `mmd_strict.xsd` file in `mmd` to initialise the config variables `mmd_xsl_path` and `mmd_xsd_path`
```

Comment:
would maybe put the full path from the root mmd folder here:
- `xslt/mmd-to-geonorge.xsl` 
- `xsd/mmd_strict.xsd`
just to be extra clear

Reply:
yes, this would be good